colosseum/constants.py

```diff
@@ -13,6 +13,12 @@ def __init__(
         self.validators = validators or []
 
     def validate(self, value):
+        try:
+            if callable(value.value):
+                return value
+        except AttributeError:
+            pass
```

Comment:
Wait - what is the use case for this check? If you're calling `choices.validate(v)`, it's to determine whether `v` is a legal value for the user to provide. A callable won't *ever* be a valid value, so why is it being accepted here? From the look of it, the underlying problem is that `validated_property` is making a simple call to `validate` at time of constructions. A callable is valid in *that* context, but not generally - the logic should reflect that.